interfaces/Portal/package.json

```
@@ -25,7 +25,8 @@
     "test:all": "npm run lint && npm test",
     "test:dev": "ng test",
     "test": "ng test --configuration=ci",
-    "upgrade:angular": "ng update @angular/core @angular/cli @angular-eslint/schematics --create-commits"
+    "upgrade:angular": "ng update @angular/core @angular/cli @angular-eslint/schematics --create-commits",
+    "postinstall": "cd ../../services/121-service && npm ci --omit=optional --no-fund --no-audit"
```

This is necessary to be able to use the full `@121-service` imports (mainly in CI). Without it, the Portal compiler will complain when we reference controllers and such, with errors such as `typeorm could not be found`.

---

Aha, because only some "magical mapping aliasses" wont work for the "built-in types" of some TypeORM-related things, right?

Feels a bit weird to 'hard-code' this install-loop in this way... But otherwise we would need to document it somewhere anyway... AND we would need to perform the correct install-steps in any automated test-workflow...

But there is still this risk of "the npm install for the 121-services will run multiple times in a test-run", I think...
And also a "build" of the Portal will require also to "install all dependencies of the 121-service". :(

---

```suggestion
    "postinstall": "cd ../../services/121-service && npm install"
```

I would let the 121-service manage its own "install-step"; so we don't have to keep any configuration-flags etc in-sync..

---

> Aha, because only some "magical mapping aliasses" wont work for the "built-in types" of some TypeORM-related things, right?

Yup

> Feels a bit weird to 'hard-code' this install-loop in this way... But otherwise we would need to document it somewhere anyway... AND we would need to perform the correct install-steps in any automated test-workflow...

Yes, my thinking exactly. This has effectively become part of the installation process of the Portal at this point.

> But there is still this risk of "the npm install for the 121-services will run multiple times in a test-run", I think...
And also a "build" of the Portal will require also to "install all dependencies of the 121-service". :(

Yes and yes, but I can't think of a way around that.

> Or, to make this "feature" more explicit to both the Portal-developers AND the 121-service-developers:

I like this suggestion 👍